index.html

```diff
@@ -1,51 +1,184 @@
+      </nav>
     </header>
     <!-- Welcome Section -->
     <section>
-        <div class="welcome-section">
-            <div class="container">
-                <h1>Hey there. I'm <br/> Zied</h1>
-            <h2>I'm a software developer</h2>
-            <p>
-                 I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-don’t hestiate to contact me.
-            </p>
-            
-            <div class="social-link">
-                <ul>
-                    <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
-                    <li><a href="#"><i class="fa-brands fa-linkedin-in"></i></a></li>
-                    <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
-                    <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li>
-                </ul>
+      <div class="welcome-section">
+        <div class="container">
+          <h1>
+            Hey there. I'm <br />
+            Zied
+          </h1>
+          <h2>I'm a software developer</h2>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+
+          <div class="social-link">
+            <ul>
+              <li>
+                <a href="#"><i class="fa-brands fa-github"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa-brands fa-linkedin-in"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa-brands fa-twitter"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa-brands fa-angellist"></i></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- Divide Arrow -->
+    <div class="bottom-arrow">
+      <div>
+        <a href="#">
+          <span class="bottom"></span>
+        </a>
+      </div>
+    </div>
+    <!-- Works Section -->
+    <section class="work-section">
+      <h2 class="works-title">My Recent Works</h2>
+      <div class="projects-container">
+        <div class="project">
+          <div class="project-image">
+            <img src="images/Asset3.png" alt="project1" />
+          </div>
+          <div class="project-description">
+            <h3>Multi-Post Stories Gain+Glory</h3>
+            <ul>
+              <li>Ruby on rails</li>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+            <a href="#" class="btn">See Project</a>
+          </div>
+        </div>
+        <div class="project">
+          <div class="project-image">
+            <img src="images/Asset3.png" alt="project1" />
```

Comment:
- [ ] There are no background images in the cards in the Figma template, please the design must be similar to the template. 😳 

- Figma tem plate 
![image](https://user-images.githubusercontent.com/58642949/185608061-bc66462a-75d0-4e46-9945-2e90580e29d1.png)

- Yours:
![image](https://user-images.githubusercontent.com/58642949/185608145-5537f240-418a-484c-844d-096929e4aa59.png)